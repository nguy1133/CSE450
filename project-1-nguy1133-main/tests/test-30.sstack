// If the delimiter is not in the pattern, SPLIT pushes the original string
// and then an empty string onto the stack.
SPLIT "delim" "This is a pattern that won't be split"
PRINT // Print a blank line due to the empty string.
PRINT // Now print "This is a pattern that won't be split".
