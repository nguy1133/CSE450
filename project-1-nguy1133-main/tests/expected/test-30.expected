
This is a pattern that won't be split
